CHANGELOG.md

```diff
@@ -7,41 +7,71 @@ CHANGELOG
 **ENHANCEMENTS**
 
 - Add support for CentOS 8 in all Commercial regions.
+- Add support for P4d instance type as compute node.
+- Add the possibilty to enable NVIDIA GPUDirect RDMA support on EFA by using the new `enable_efa_gdr` configuration
+  parameter.
 - Enable support for NICE DCV in GovCloud regions.
 - Enable support for AWS Batch scheduler in GovCloud regions.
-- Add support for P4d instance type as compute node.
-- Add ``-r/-region`` arg to ``pcluster configure``. If this arg is provided, configuration will skip region selection.
-- Add `cluster_resource_bucket` parameter under `cluster` section of cluster config to allow using user-provided bucket 
-  for hosting S3 resources used by cluster.
+- FSx Lustre:
+  - Add possibility to configure Auto Import policy through the new `auto_import_policy` parameter.
+  - Add support to HDD storage type and the new `storage_type` and `drive_cache_type` configuration parameters.
+- Create a CloudWatch Dashboard for the cluster, named `<clustername>-<region>`, including head node EC2 metrics and 
+  cluster logs. It can be disabled by configuring the `enable` parameter in the `dashboard` section.
+- Add `-r/-region` arg to `pcluster configure` command. If this arg is provided, configuration will 
+  skip region selection.
+- Add `-r/-region` arg to`ssh` and `dcv connect` commands.
+- Add `cluster_resource_bucket` parameter under `cluster` section to allow the use of a user-provided S3 bucket.
```

Comment:
Add `cluster_resource_bucket` parameter under `cluster` section to allow the user to specify an existing S3 bucket.